gnomad/resources/resource_utils.py

```diff
@@ -395,17 +395,23 @@ def read_resource(self, *args, **kwargs):
                 # If one of the known sources is selected, check if the resource is available.
                 # For custom sources, skip the check and attempt to read the resource.
                 resource_source = gnomad_public_resource_configuration.source
-                if (
-                    isinstance(resource_source, GnomadPublicResourceSource)
-                    and resource_source != GnomadPublicResourceSource.GNOMAD
-                ):
-                    if not self.is_resource_available():
-                        raise ResourceNotAvailable(
-                            f"This resource is not currently available from {resource_source.value}.\n\n"
-                            "To load resources directly from gnomAD instead, use:\n\n"
-                            ">>> from gnomad.resources.config import gnomad_public_resource_configuration, GnomadPublicResourceSource\n"
-                            ">>> gnomad_public_resource_configuration.source = GnomadPublicResourceSource.GNOMAD"
-                        )
+                if not self.is_resource_available():
+                    if resource_source == GnomadPublicResourceSource.GNOMAD:
```

Comment:
Just making sure I understand, if the resource_source is 'gnomAD' rather than `Google Cloud Public Datasets` then state that `This resource is not currently available from the gnomAD project.`. Maybe we should say `This resource is not currently available from the gnomAD project public buckets.` Or something that indicates we don't host it, but it is still in other locations that host gnomAD project data